nova/compute.sls

```diff
@@ -143,8 +143,21 @@ Add_compute_to_availability_zone_{{ compute.availability_zone }}:
   - name: "nova {{ identity_params }} aggregate-add-host {{ compute.availability_zone }} {{ pillar.linux.system.name }}"
   - unless: "nova {{ identity_params }} service-list | grep {{ compute.availability_zone }} | grep {{ pillar.linux.system.name }}"
 
+{%- if compute.aggregates != None %}
+
+{%- for aggregate in compute.aggregates %}
+Add_compute_to_aggregate_{{ aggregate }}:
```

Comment:
Nah. I've tested it on working env. Will have a chance to deploy it from
scratch next week.